src/Symfony/Resources/config/services/ezplatform.yaml

```diff
@@ -38,3 +38,8 @@ services:
             $views: { field: '%ezcontentquery_field_view%', item: '%ezcontentquery_item_view%' }
         tags:
             - { name: kernel.event_subscriber }
+
+    ezpublish.fieldType.indexable.unindexed:
+        class: eZ\Publish\Core\FieldType\Unindexed
+        tags:
+            - {name: ezplatform.field_type.indexable, alias: ezcontentquery}
```

Comment:
Missing line feed at the end of file (seen in github /files list)

Reply:
Funny note: we tag as "indexable" a class named "Unindexable". I like that.